flake.nix

```diff
@@ -1,14 +1,32 @@
 {
   description = "provider proxy";
 
-  inputs.flake-utils.url = "github:numtide/flake-utils";
   inputs.nixpkgs.url = github:NixOS/nixpkgs/nixpkgs-unstable;
```

Comment:
Should we not prefer stable?

Reply:
unstable contains the recent wasm-pack etc